exercises/faq-page/README.md

```diff
@@ -0,0 +1,112 @@
+# Frequently Asked Questions and Cheat Sheet
+
+> This section contains common information needed when working through the exercises and a command line cheat sheet to help with completing exercises.
+
+---
+
+## Frequently Asked Questions
+
+- How can I get into my CodeReady Workspace (assumes you have gone through Exercise 1, Part 1 where we created the cloud IDE)?
+
+  1. Navigate to the CodeReady workspaces link provided by your facilitators. Take note that this is not the exact same link as in Exercise 1, Part 1 which was used to deploy the cloud IDE. After navigating to the link, you should see a page similar to the picture below.
+
+  ![code-ready-workspaces](../images/faq/codeready-workspace-entrypoint.png)
+
+  2. On the left nav bar under "Recent Workspaces", select the workspace that has `do500-id` (in the picture above, that's `do500-p4ez1`).
+
+  3. The workspace should then start up and when complete you should be back in the cloud IDE.
+
+- How do I get the OpenShift login token?
+
+  1. Login to the cluster at the cluster URL provided.
+
+  2. At the top right of the page after logging in, you should see your name. Click into that tab and under your name should be a `Copy Login Command` url that you should click.
+
+  ![ocp-login-token-link](../images/faq/ocp-token-link.png)
+
+  3. You should then be prompted for your credentials. Click `ldap` and login using your cluster credentials.
```

Comment:
Where were the cluster credentials provided?  I know that this seems picky but they get a bunch of mails, invites, stuff written on boards and they all have terms in that that the users are unfamliar with so can be confusing...